packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3320,6 +3320,7 @@ class InputDecoration {
     int errorMaxLines,
     bool hasFloatingPlaceholder,
     FloatingLabelBehavior floatingLabelBehavior,
+    bool isCollapsed,
```

Comment:
The doc comment is no longer accurate.

Reply:
Thanks for catching that!  It makes me worry about people relying on the old behavior, though I think technically it's not a breaking change.  I wonder why isCollapsed was not passed through in the first place.